Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -179,13 +197,19 @@ def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetche
         total_fetched (int): The amount of indicators already fetched
         next_page (int): The page we are up to in the loop
         last_found_len (int): The amount of indicators found in the last fetch
+        use_legacy_query (bool): Whether to use filter fields in the indicators search
 
     Returns:
         (tuple): The iocs and the last page
     """
     iocs: List[dict] = []
-    filter_fields = "name,type"  # based on func ToIoC https://github.com/demisto/server/blob/master/domain/insight.go
-    search_indicators = IndicatorsSearcher(page=next_page, filter_fields=filter_fields)
+    indicators_searcher_args: Dict[str, Union[int, str]] = {
```

Comment:
I am not a big fan of using here a dict. It is less readable and error prone. I would do something of the form:
```python
filter_fields = 'name,type' if not use_legacy_query else None
search_indicators = IndicatorsSearcher(page=next_page, filter_fields=filter_fields)
```